### DISCUSSION TITLE
Highlight unused variables in DAX code (feature request)

### DISCUSSION THREAD

**Discussion by avatorl** [TabularEditor/TabularEditor3#1055] [Ideas]
It would be great to have a feature that highlight unused variables in DAX code. By unused variables I mean variables that are not included into a final RETURN statement directly or via other variables.

VAR a = 1
VAR b = 2
_VAR c = 3_
VAR d = a + b
RETURN d

Highlight variable c.

**Comment by otykier** [comment]
Great suggestion!

**Comment by marcosqlbi** [comment]
I like it too!

**Comment by otykier** [comment]
Now available in [3.12.0](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.12.0), thanks for the suggestion!
